test/test_testing.py

```diff
@@ -574,11 +574,10 @@ def test_unknown_layout(self):
 
     def test_meta(self):
         actual = torch.empty((2, 2), device="meta")
-        expected = actual.clone()
+        expected = torch.empty((2, 2), device="meta")
```

Comment:
tbh I'm not sure what this test is supposed to be testing anymore XD I suppose this is fine...